Make file does not work, Pop!_OS jammy 22.04 x86_64

make -C nautilus_open_any_terminal/locale
make[1]: Entering directory '/home/myname/GitClone/nautilus-open-any-terminal/nautilus_open_any_terminal/locale'
msgfmt -o cs.mo cs.po
make[1]: msgfmt: No such file or directory
make[1]: *** [Makefile:12: cs.mo] Error 127
make[1]: Leaving directory '/home/myname/GitClone/nautilus-open-any-terminal/nautilus_open_any_terminal/locale'
make: *** [Makefile:22: build] Error 2

--- a/README.md
+++ b/README.md
@@ -100,6 +100,8 @@ pip install nautilus-open-any-terminal
 ```
 
 ### From source
+Requires `gettext`.
+
 ```sh
 git clone https://github.com/Stunkymonkey/nautilus-open-any-terminal.git
 cd nautilus-open-any-terminal

